cypress/integration/predictions/predictions.test.ts

```diff
@@ -1,12 +0,0 @@
-describe('Predictions Page', () => {
-  it('User can get past to disclaimer', () => {
-    cy.visit('/prediction')
-    cy.get('#predictions-risk-disclaimer').should('be.visible')
-    cy.get('#prediction-disclaimer-continue').should('be.disabled')
-    cy.get('#responsibility-checkbox').click()
-    cy.get('#beta-checkbox').click()
-    cy.get('#prediction-disclaimer-continue').should('not.be.disabled')
-    cy.get('#prediction-disclaimer-continue').click()
-    cy.get('#predictions-risk-disclaimer').should('not.exist')
-  })
-})
```

Comment:
is this test case not suited anymore?
why should we remove this

Reply:
Yup. Because we use `DisclaimerModal`, so no have so much or id can be check.

Reply:
we could remove the test case or we can pass the id to props and make the id dynamic, so we can still have the test case to protect.

what do you guys think @ChefMomota @chef-jojo @chefilip

Reply:
@Chef-Yogi  fixed